### ISSUE TITLE
After upgrade, eslint no longer works, please fix

### ISSUE THREAD

**Issue by JoshuaVSherman** [WebJamApps/web-jam-back#367]
yarn run v1.16.0
$ eslint . --fix
Error: Failed to load plugin 'jsx-a11y' declared in '.eslintrc.json » eslint-config-airbnb » /home/joshsherman/github/web-jam-back/node_modules/eslint-config-airbnb/rules/react-a11y.js': Cannot find module '@babel/runtime/helpers/interopRequireDefault'
Require stack:
- /home/joshsherman/github/web-jam-back/node_modules/eslint-plugin-jsx-a11y/lib/rules/accessible-emoji.js
- /home/joshsherman/github/web-jam-back/node_modules/eslint-plugin-jsx-a11y/lib/index.js
- /home/joshsherman/github/web-jam-back/node_modules/eslint/lib/cli-engine/config-array-factory.js
- /home/joshsherman/github/web-jam-back/node_modules/eslint/lib/cli-engine/cascading-config-array-factory.js
- /home/joshsherman/github/web-jam-back/node_modules/eslint/lib/cli-engine/cli-engine.js
- /home/joshsherman/github/web-jam-back/node_modules/eslint/lib/cli-engine/index.js
- /home/joshsherman/github/web-jam-back/node_modules/eslint/lib/cli.js
- /home/joshsherman/github/web-jam-back/node_modules/eslint/bin/eslint.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:625:15)
    at Function.Module._load (internal/modules/cjs/loader.js:527:27)
    at Module.require (internal/modules/cjs/loader.js:683:19)
    at require (internal/modules/cjs/helpers.js:16:16)
    at Object.<anonymous> (/home/joshsherman/github/web-jam-back/node_modules/eslint-plugin-jsx-a11y/lib/rules/accessible-emoji.js:3:30)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:643:32)
    at Function.Module._load (internal/modules/cjs/loader.js:556:12)
    at Module.require (internal/modules/cjs/loader.js:683:19)
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
